es-vue-base/src/lib-components/EsErrorPage.vue

```diff
@@ -1,21 +1,16 @@
 <template>
-    <b-container class="my-450">
+    <b-container class="my-450 bg-gray-50">
         <b-row>
             <b-col
                 md="7"
                 class="my-450 mx-auto text-center">
-                <h1 class="font-weight-boldest display-3">
-                    {{ errorType }}
+                <h1 class="font-weight-bolder display-3 text-gray-600">
+                    {{ errorShortMessage }}
                 </h1>
                 <h2
```

Comment:
This was pre-existing, but semantically, this content should be a `<p>`, not an `<h2>` tag, as it's not a heading for content underneath it - can we update this as well